src/engine/renderer/tr_local.h

```diff
@@ -3093,7 +3099,7 @@ inline bool checkGLErrors()
 
 	void           R_AddPolygonSurfaces();
 
-	void           R_AddDrawSurf( surfaceType_t *surface, shader_t *shader, int lightmapNum, int fogNum, bool bspSurface = false );
+	int           R_AddDrawSurf( surfaceType_t *surface, shader_t *shader, int lightmapNum, int fogNum, bool bspSurface = false );
```

Comment:
Same (and in fact alignment is already lost, so better strip that space).

Reply:
Also done now.